newsfragments/44908.significant.rst

```diff
@@ -0,0 +1,15 @@
+Airflow CLI command ``airflow config lint`` added
```

Comment:
Not sure whether we really need a `significant` (breaking-change) newsfragment for this. more like a feature for me. 🤔

Reply:
Refactored to add it as feature